numba/_dispatcher.c

```diff
@@ -6,6 +6,83 @@
 
 #include "_dispatcher.h"
 #include "_typeof.h"
+#include "frameobject.h"
+
+#define PROFILE_CALL 1
```

Comment:
Right, I will remove it. (It was a simple means to switch between the "old" and the "new" code.)